src/js/Icon.js

```diff
@@ -1,31 +1,39 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import deepAssign from 'deep-assign';
-import { CHANNEL } from 'styled-components/lib/models/ThemeProvider';
+import {
+  CHANNEL_NEXT,
+  CONTEXT_CHANNEL_SHAPE,
+} from 'styled-components/lib/models/ThemeProvider';
```

Comment:
Shouldn't this be `styled-components/src/models/ThemeProvider` with `src` instead of `lib`? I'm debugging a weird issue that showed up since upgrading `grommet-icons` and it points to this being an invalid location.